python/plugins/processing/ui/resultsdockbase.ui

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>DockWidget</class>
+ <widget class="QDockWidget" name="DockWidget">
```

Comment:
QgsDockWidget is in qgis.gui, so you'll need to change the PyQt5.QtWidgets over to qgis.gui in the .ui file.